test/functional/feature_rbf.py

```diff
@@ -28,14 +28,17 @@ def set_test_params(self):
         self.num_nodes = 2
         self.extra_args = [
             [
+                "-mempoolfullrbf=0",
```

Comment:
Might be better to add a comment above line 29 a la
```
# both nodes disable full-rbf to test BIP125 signaling
```

and then change the comment on line 38 to
```
# second node has default mempool limits
```

Reply:
Thanks! Good idea; fixed.